platform/ios/src/MGLPinAnnotationView.m

```diff
@@ -0,0 +1,72 @@
+#import "MGLPinAnnotationView.h"
+
+@implementation MGLPinAnnotationView
+
+- (instancetype)initWithReuseIdentifier:(nullable NSString *)reuseIdentifier {
+    self = [super initWithReuseIdentifier:reuseIdentifier];
+    _shadowColor = [UIColor blackColor];
+    _pinColor = [UIColor blueColor];
+    _strokeColor = [UIColor whiteColor];
+    _innerColor = [UIColor whiteColor];
+
+    return self;
+}
+
+- (void)drawRect:(CGRect)rect {
+    
+    //// General Declarations
```

Comment:
If this code was generated by a graphical tool, please commit the original file under misc/.